scripts/qa_crowbarsetup.sh

```diff
@@ -3258,6 +3258,7 @@ function deploy_single_proposal()
         magnum)
             [[ -n "$want_magnum" ]] || return
             if iscloudver 7plus ; then
+                get_novacontroller
```

Comment:
Why is it working here, then? https://ci.suse.de/job/openstack-mkcloud/32829/console